From dd1126992313fc6cb7bb5f7379c3886e0922d657 Mon Sep 17 00:00:00 2001
From: Evan Wang <xswang@marvell.com>
Date: Wed, 30 Aug 2017 19:21:48 +0800
Subject: [PATCH 1226/1345] Revert "dts: pcie: a70x0: a80x0: add comphy handle
 to PCIe node"

commit  a78ab46f49eed38baba3a09f132c9d5014141397 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

This reverts commit c4970954390e
("dts: pcie: a70x0: a80x0: add comphy handle to PCIe node")

After introducing PCIe driver, SATA over PCIe does not work
well any more, and before finding the root cause for the
issue, the PCIe COMPHY driver will be disbaled via
reverting the patch of "dts: pcie: a70x0: a80x0: add
comphy handle to PCIe node".
Once the issue fixed, this patch will be reverted.

Change-Id: I2926b1be41eee560690a7df2a122689deb2771df
Signed-off-by: Evan Wang <xswang@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/43550
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db-B.dts   |    8 --------
 arch/arm64/boot/dts/marvell/armada-7040-db-C.dts   |    4 ----
 arch/arm64/boot/dts/marvell/armada-7040-db-D.dts   |    4 ----
 arch/arm64/boot/dts/marvell/armada-7040-db-E.dts   |    4 ----
 .../arm64/boot/dts/marvell/armada-7040-db-nand.dts |    4 ----
 arch/arm64/boot/dts/marvell/armada-7040-db.dts     |    2 --
 arch/arm64/boot/dts/marvell/armada-7040-pd.dts     |    2 --
 arch/arm64/boot/dts/marvell/armada-8040-db-B.dts   |   14 --------------
 arch/arm64/boot/dts/marvell/armada-8040-db-C.dts   |   10 ----------
 arch/arm64/boot/dts/marvell/armada-8040-db-D.dts   |   10 ----------
 arch/arm64/boot/dts/marvell/armada-8040-db-E.dts   |   10 ----------
 arch/arm64/boot/dts/marvell/armada-8040-db.dts     |   10 ----------
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts  |    6 ------
 13 files changed, 88 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
index c7a0cfd..2158221 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-B.dts
@@ -174,20 +174,12 @@
 		pcie@0x600000 {
 			status = "okay";
 			num-lanes = <4>;
-			phys = <&comphy0 0 COMPHY_PCIE0
-				&comphy0 1 COMPHY_PCIE0
-				&comphy0 2 COMPHY_PCIE0
-				&comphy0 3 COMPHY_PCIE0
-				>;
-			phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 		};
 		pcie@0x620000 {
 			status = "disabled";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
index 60f219f..d53681a 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-C.dts
@@ -200,16 +200,12 @@
 
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x620000{
 			status = "disabled";
 		};
 		pcie@0x600000{
 			status="okay";
-			phys = <&comphy0 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
index 73beaa1..2229329 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-D.dts
@@ -204,16 +204,12 @@
 
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy0 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x620000 {
 			status = "disabled";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
index e151717..1dc199a 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-E.dts
@@ -172,16 +172,12 @@
 
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x620000 {
 			status = "disabled";
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy0 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db-nand.dts b/arch/arm64/boot/dts/marvell/armada-7040-db-nand.dts
index 635d282..040f237 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db-nand.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db-nand.dts
@@ -180,13 +180,9 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy0 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index d1fbfad..ca3dd55 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -176,8 +176,6 @@
 
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 
diff --git a/arch/arm64/boot/dts/marvell/armada-7040-pd.dts b/arch/arm64/boot/dts/marvell/armada-7040-pd.dts
index b6fbfdf..fd44018 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-pd.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-pd.dts
@@ -272,8 +272,6 @@
 
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
index 6d2d1a0..a64f17f 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-B.dts
@@ -151,17 +151,9 @@
 		pcie@0x600000 {
 			status = "okay";
 			num-lanes = <4>;
-			phys = <&comphy0 0 COMPHY_PCIE0
-				&comphy0 1 COMPHY_PCIE0
-				&comphy0 2 COMPHY_PCIE0
-				&comphy0 3 COMPHY_PCIE0
-				>;
-			phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 	cpn-110-slave {
@@ -234,18 +226,12 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy1 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x620000 {
 			status = "okay";
-			phys = <&comphy1 4 COMPHY_PCIE1>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy1 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
index 25efd58..e5d44e4 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-C.dts
@@ -165,13 +165,9 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy0 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 	cpn-110-slave {
@@ -259,18 +255,12 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy1 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x620000 {
 			status = "okay";
-			phys = <&comphy1 4 COMPHY_PCIE1>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy1 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
index 87978d1..298ebbf 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-D.dts
@@ -166,13 +166,9 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy0 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 	cpn-110-slave {
@@ -236,18 +232,12 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy1 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x620000 {
 			status = "okay";
-			phys = <&comphy1 4 COMPHY_PCIE1>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy1 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
index 4c722e8..0fe4ac4 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-E.dts
@@ -168,13 +168,9 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy0 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 	cpn-110-slave {
@@ -251,18 +247,12 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy1 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x620000 {
 			status = "okay";
-			phys = <&comphy1 4 COMPHY_PCIE1>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy1 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dts b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
index 216bf25..1920819 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
@@ -122,13 +122,9 @@
 		};
 		pcie@0x600000 {
 			status = "okay";
-			phys = <&comphy0 0 COMPHY_PCIE0>;
-			phy-names = "pcie-phy0";
 		};
 		pcie@0x640000 {
 			status = "okay";
-			phys = <&comphy0 5 COMPHY_PCIE2>;
-			phy-names = "pcie-phy0";
 		};
 
 		cpm_usb3h0_phy: cpm_usb3_phy0 {
@@ -488,16 +484,10 @@
 
 &cps_pcie0 {
 	status = "okay";
-	phys = <&comphy1 0 COMPHY_PCIE0>;
-	phy-names = "pcie-phy0";
 };
 &cps_pcie1 {
 	status = "okay";
-	phys = <&comphy1 4 COMPHY_PCIE1>;
-	phy-names = "pcie-phy0";
 };
 &cps_pcie2 {
 	status = "okay";
-	phys = <&comphy1 5 COMPHY_PCIE2>;
-	phy-names = "pcie-phy0";
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index 5bf7aee..94cb12a 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -132,12 +132,6 @@
 		pcie@0x600000 {
 			num-lanes = <4>;
 			status = "okay";
-			phys = <&comphy0 0 COMPHY_PCIE0
-				&comphy0 1 COMPHY_PCIE0
-				&comphy0 2 COMPHY_PCIE0
-				&comphy0 3 COMPHY_PCIE0
-				>;
-			phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 			reset-gpios = <&cpm_gpio1 20 GPIO_ACTIVE_HIGH>;
 			pinctrl-names = "default";
 			pinctrl-0 = <&pcie_reset_pins>;
-- 
1.7.9.5

